keras_nlp/models/whisper/whisper_presets.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+LANGUAGE_TOKENS = {
+    "<|af|>": 50327,
```

Comment:
This is a big long list to keep around. Given that we are already passing in a special token json file, maybe let's keep this as a json file on the preset too. Newer presets. could then support more language codes in theory.